src/Bicep.LangServer/Snippets/Templates/res-mg.bicep

```diff
@@ -0,0 +1,13 @@
+// Management Group

```

Comment:
Add comment to the description about the target scope being tenant (is it only tenant?  if no, please modify):

// Management Group (for tenant scope)